mock/environment/__init__.py

```diff
@@ -16,34 +17,49 @@
 class Env:
    
     def __init__(self):
+        """
+        Stores weather data into a dictionary 
+        that can be indexed by site and date.
+        """
```

Comment:
Instead of reusing `'Time_Stamp_UTC'` over and over again, which makes it prone to typos, use a variable instead.

Same with :
```
day_difference = timedelta(hours=7)
```
so that we can change this easily.